changed pip spawn locations

HaydnNeese opened the pull request.

HaydnNeese committed:
changed pip spawn locations

lancehalberd commented on src/renderDigging.js:
```diff
@@ -423,14 +423,15 @@ function getPipPoints(state, row, column) {
-        [x + HEX_WIDTH - SHORT_EDGE - pad, y + ROW_HEIGHT / 3 + pad],
+        [x + HEX_WIDTH / 4 + HEX_WIDTH / 4 * c, y + ROW_HEIGHT / 2 * c],
+        [x + 3 * HEX_WIDTH / 4 - HEX_WIDTH / 4 * c, y + ROW_HEIGHT / 2 * c],
+        [x + SHORT_EDGE, y + ROW_HEIGHT / 2],
```
These two center ones should probably use `c` as well, maybe:
`x + HEX_WIDTH / 2 * c`?

lancehalberd reviewed (changes requested):
Once you've applied the changes, it would be good to take a couple of screenshots that demonstrate the location of all 7 pips. I forgot to mention that earlier, but a good practice for anything that effects the UI is to always add screenshots to the PR request. It is especially good if you can show before and after for what is changed, but you don't need to worry as much about the before shots.

This way a reviewer won't have to pull your branch and run the code if they are curious what things look like with the changes.

HaydnNeese committed:
Update renderDigging.js

lines 430 and 432
[x + SHORT_EDGE * c, y + ROW_HEIGHT / 2 * c],
and
[x + HEX_WIDTH - SHORT_EDGE, y + ROW_HEIGHT / 2],

HaydnNeese committed:
Update renderDigging.js

HaydnNeese reviewed (commented):
I applied the c variable to both of those lines for you

lancehalberd commented on src/renderDigging.js:
```diff
@@ -423,14 +423,15 @@ function getPipPoints(state, row, column) {
+        [x + HEX_WIDTH / 4 + HEX_WIDTH / 4 * c, y + ROW_HEIGHT / 2 * c],
+        [x + 3 * HEX_WIDTH / 4 - HEX_WIDTH / 4 * c, y + ROW_HEIGHT / 2 * c],
+        [x + SHORT_EDGE * c, y + ROW_HEIGHT / 2 * c],
+        [x + HEX_WIDTH * c/ 2, y + ROW_HEIGHT / 2 * c],
```
This line shouldn't have `c`, this is the center dot, and is always in the same position.

lancehalberd replied:
Also, did you want to add those screenshots, or should I just go ahead and merge this in?

HaydnNeese replied:
oh sorry, I thought you meant on future merges do screenshots.
And I did it right on the Visual Code I made the change separately on Github and should've just copy and pasted.

lancehalberd commented on src/renderDigging.js:
```diff
@@ -423,14 +423,15 @@ function getPipPoints(state, row, column) {
-        [x + HEX_WIDTH / 2, y + 5 * ROW_HEIGHT / 6 - pad * 1.5],
+        [x + HEX_WIDTH / 4 + HEX_WIDTH / 4 * c, y + ROW_HEIGHT / 2 * c],
+        [x + 3 * HEX_WIDTH / 4 - HEX_WIDTH / 4 * c, y + ROW_HEIGHT / 2 * c],
+        [x + SHORT_EDGE * c, y + ROW_HEIGHT / 2 * c],
```
This (and on 432) should probably be `EDGE_LENGTH` or `HEX_WIDTH / 2` instead of `SHORT_EDGE`.

HaydnNeese committed:
Update renderDigging.js

Is this correct?

HaydnNeese committed:
Removed the unnecessary locations of 'c'

lancehalberd reviewed (approved):
Looks good now!
